tutorials/rendering/renderers.rst

```diff
@@ -0,0 +1,357 @@
+| features            |                          |                          |                          |
++---------------------+--------------------------+--------------------------+--------------------------+
+| Core 3D rendering   | ✔️ Yes.                  | ✔️ Yes.                  | ✔️ Yes.                  |
+| features            |                          |                          |                          |
++---------------------+--------------------------+--------------------------+--------------------------+
+| Advanced            | ❌ No.                   | ⚠️ Yes, limited by       | ✔️ Yes. All rendering    |
+| rendering features  |                          | mobile hardware.         | features are supported.  |
++---------------------+--------------------------+--------------------------+--------------------------+
+| New features        | ⚠️ Some new rendering    | ✔️ Most new rendering    | ✔️ All new features are  |
+|                     | features are added to    | features are added to    | added to Forward+. As the|
+|                     | Compatibility. Features  | Mobile. Mobile usually   | focus of new development,|
+|                     | are added after Mobile   | gets new features as     | Forward+ gets features   |
+|                     | and Forward+.            | Forward+ does.           | first.                   |
++---------------------+--------------------------+--------------------------+--------------------------+
+| Rendering cost      | Low base cost, but       | Medium base cost, and    | Highest base cost, and   |
+|                     | high scaling cost.       | medium scaling cost.     | low scaling cost.        |
++---------------------+--------------------------+--------------------------+--------------------------+
+| Rendering driver    | OpenGL.                  | Vulkan, Direct3D 12, or  | Vulkan, Direct3D 12, or  |
+|                     |                          | Metal.                   | Metal.                   |
++---------------------+--------------------------+--------------------------+--------------------------+
+
+Lights and shadows
+~~~~~~~~~~~~~~~~~~
+
+See :ref:`doc_lights_and_shadows` for more information.
+
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Feature                 | Compatibility            | Mobile                   | Forward+                 |
++=========================+==========================+==========================+==========================+
+| Lighting approach       | Forward                  | Forward                  | Clustered Forward        |
+|                         |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Maximum                 | 8 per mesh. Can be       | 8 per mesh, 256 per view.| Unlimited.               |
```

Comment:
I don't think it is true that the Forward+ renderer supports Unlimited lights. 

The clustered element limit is per-clustered item i.e. ( the limit is 512 omni lights, 512 spot lights, 512 reflectionprobes, 512 decals, per cluster)

I would remove the clustered elements line, and instead record it here:
"512 per cluster. Can be increased."